### PR TITLE
add 2 params on each importer

### PR DESCRIPTION
**Pull request by antoine-de**
linked to https://github.com/CanalTP/mimirsbrunn/pull/268

add the possibility to control the number of shards/replicas of each ES
index.

### CODE REVIEW

File: tasks.py [hunk 1]
```diff
@@ -94,19 +94,30 @@ def generate_cosmogony(ctx, files=[]):
...
+        --dataset {ctx.dataset} \
+        {additional_params} \
+        ".format(
+            ctx=ctx, additional_params=additional_params
         ),
     )
 
 
+def _get_cli_param(conf_value, cli_param_name):
+    if conf_value:
```
Comment (amatissart) [inline tasks.py]: Here the value 0 will be ignored.

File: tasks.py [hunk 2]
```diff
@@ -117,11 +128,19 @@ def load_osm(ctx, files=[]):
...
-    if poi_conf and "osm" in poi_conf:
-        import_poi = "--import-poi"
+    poi_conf = ctx.get("poi", {}).get('osm')
+    poi_conf = ""
+    if poi_conf:
+        poi_conf = "--import-poi"
+        poi_conf += _get_cli_param(poi_conf.get("nb_shards"), "--nb-poi-shards")
+        poi_conf += _get_cli_param(poi_conf.get("nb_replicas"), "--nb-poi-replicas")
+
+    stret_conf = ""
```
Comment (amatissart) [inline tasks.py]: `stret_conf` :arrow_right: `street_conf`

**Comment by antoine-de** [comment]
the PR has been updated (but github has some issues right now, and is not displaying it)

**Comment by antoine-de** [comment]
the PR has been updated (but github has some issues right now, and is not displaying it)

**Comment by antoine-de** [comment]
the PR has been updated (but github has some issues right now, and is not displaying it)

**Pull request merged**